<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scribbler</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTextEdit" name="textEdit"/>
      <widget class="SvgView" name="svgView"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>669</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCreate_Font"/>
    <addaction name="actionLoad_Font"/>
    <addaction name="actionRender"/>
    <addaction name="actionSave_Current_Sheet_as"/>
    <addaction name="actionPrint_Current_Sheet"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionShow_ToolBar"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout_Scribbler"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Show ToolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_Font">
   <property name="text">
    <string>Load Font</string>
   </property>
  </action>
  <action name="actionCreate_Font">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Create Font</string>
   </property>
  </action>
  <action name="actionPrint_Template">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Print Template</string>
   </property>
  </action>
  <action name="actionSave_Current_Sheet_as">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save Current Sheet as Image</string>
   </property>
  </action>
  <action name="actionSheetSize">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Sheet Size</string>
   </property>
  </action>
  <action name="actionMargins">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Margins</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>How To Use</string>
   </property>
  </action>
  <action name="actionAbout_Scribbler">
   <property name="text">
    <string>About Scribbler</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="text">
    <string>License</string>
   </property>
  </action>
  <action name="actionRender">
   <property name="text">
    <string>Render</string>
   </property>
  </action>
  <action name="actionPrint_Current_Sheet">
   <property name="text">
    <string>Print Current Sheet</string>
   </property>
  </action>
  <action name="actionShow_ToolBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show ToolBar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SvgView</class>
   <extends>QGraphicsView</extends>
   <header>svgview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
